src/main/java/pl/simpleascoding/tutoringplatform/domain/user/User.java

```diff
@@ -45,6 +46,10 @@ public class User implements UserDetails {
     @JoinColumn(name = "author_id")
     private List<Review> postedReviews = new ArrayList<>();
 
+    @OneToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REMOVE}, orphanRemoval = true)
+    @JoinColumn(name = "advertisement_id")
```

Comment:
Shouldn't `@JoinColumn` define how the user id is linked to the advertisement? To me, this line looks like it's setting the `user_id` column in the DB to be named `advertisement_id` instead, please check if this happens and correct if necessary.

Reply:
you're right